<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FORNISSEUR_P</class>
 <widget class="QMainWindow" name="FORNISSEUR_P">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FORNISSEUR_P</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>781</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color:#4F6D7A;</string>
    </property>
    <property name="title">
     <string>Gestion des fournisseur</string>
    </property>
    <widget class="QTabWidget" name="tabwidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>741</width>
       <height>621</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#C0D6DF;</string>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tab">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <attribute name="title">
       <string>ajouter</string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit_nom">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFFF;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_cin">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>140</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFFF;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_pro">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>200</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFFF;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="AJOUTER">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>150</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFF;</string>
       </property>
       <property name="text">
        <string>AJOUTER</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoRepeatDelay">
        <number>300</number>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>150</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFFF;</string>
       </property>
       <property name="text">
        <string>ANNULER</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;prenom&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <property name="indent">
        <number>0</number>
       </property>
       <property name="openExternalLinks">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>40</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:600; color:#000000;&quot;&gt;stock&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>26</x>
         <y>30</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;nom&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;cin&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_st">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFFF;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;produit&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_pr">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>90</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFFF;</string>
       </property>
      </widget>
      <zorder>lineEdit_nom</zorder>
      <zorder>lineEdit_cin</zorder>
      <zorder>lineEdit_pro</zorder>
      <zorder>pushButton_2</zorder>
      <zorder>label_7</zorder>
      <zorder>label_8</zorder>
      <zorder>label_9</zorder>
      <zorder>AJOUTER</zorder>
      <zorder>label_16</zorder>
      <zorder>lineEdit_st</zorder>
      <zorder>label_20</zorder>
      <zorder>lineEdit_pr</zorder>
     </widget>
     <widget class="QWidget" name="tab_2">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <attribute name="title">
       <string>modifier</string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit_N_nom">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFFF;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_N_cin">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>140</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFFF;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_N_produit">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>200</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFFF;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;nom&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;cin&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="MODIFIER">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>330</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFF;</string>
       </property>
       <property name="text">
        <string>MODIFIER</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoRepeatDelay">
        <number>300</number>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_11">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>330</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFFF;</string>
       </property>
       <property name="text">
        <string>ANNULER</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>40</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:600; color:#000000;&quot;&gt;stock&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;prenom&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <property name="indent">
        <number>0</number>
       </property>
       <property name="openExternalLinks">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_N_stock">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFFF;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;produit&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_N_prenom">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>90</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFFF;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>340</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;cin&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_CIN1">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>340</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFFF;</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <attribute name="title">
       <string>supprimer</string>
      </attribute>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>176</x>
         <y>90</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;cin&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_cin_S">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>90</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFFF;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="SUPPRIMER">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>180</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFF;</string>
       </property>
       <property name="text">
        <string>SUPPRIMER</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoRepeatDelay">
        <number>300</number>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_13">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>180</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFFF;</string>
       </property>
       <property name="text">
        <string>ANNULER</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <attribute name="title">
       <string>afficher</string>
      </attribute>
      <widget class="QTableView" name="tab_FOURNISSEUR">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>601</width>
         <height>261</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFFF;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>290</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFFF;</string>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>290</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFFF;</string>
       </property>
       <property name="text">
        <string>STATS</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_9">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>320</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFFF;</string>
       </property>
       <property name="text">
        <string>sms</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_10">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>320</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFFF;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_recherche">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>350</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>360</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>RECHERCHER</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>280</y>
         <width>112</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>trier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="xcl">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>360</y>
         <width>112</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>xcl</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Chat</string>
      </attribute>
      <widget class="QTableView" name="tab_message">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>311</width>
         <height>271</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_envoyer">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>380</y>
         <width>112</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>Envoyer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>120</y>
         <width>69</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>NOM</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id_chat">
       <property name="geometry">
        <rect>
         <x>442</x>
         <y>110</y>
         <width>161</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_message">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>170</y>
         <width>181</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>190</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Message</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Mailer</string>
      </attribute>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>585</width>
         <height>551</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QFrame" name="frame_50">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_31">
           <item>
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>Smtp-server:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="server">
             <property name="text">
              <string>smtp.gmail.com</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>Server port:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="port">
             <property name="text">
              <string>465</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>Username:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="uname">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string>Password:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="paswd">
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_29">
             <property name="text">
              <string>Recipant to:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="rcpt"/>
           </item>
           <item>
            <widget class="QLabel" name="label_30">
             <property name="text">
              <string>Subject:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="subject"/>
           </item>
           <item>
            <widget class="QLabel" name="label_31">
             <property name="text">
              <string>Message:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="msg"/>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>545</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="sendBtn">
             <property name="styleSheet">
              <string notr="true"> QPushButton {
color: #ffffff;
border: 3px solid #f8d466;
font: 8pt &quot;MS Reference Sans Serif&quot;;
border-radius: 15px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #aa3118, stop: 1 #550000);
min-width: 80px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #aa3118, stop: 1 #552a2a);
}

 QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 3, stop: 0 #aa3118, stop: 1 #554040);
}</string>
             </property>
             <property name="text">
              <string>Send</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>907</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
